sagemaker-python-sdk/keras_script_mode_pipe_mode_horovod/tensorflow_keras_CIFAR10.ipynb

```diff
@@ -0,0 +1,618 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Train and Host a Keras Sequential Model\n",
+    "## Utilizing Pipe Mode datasets and distributed training with Horovod\n",
+    "This notebook shows how to train and host a Keras Sequential model on SageMaker. The model used for this notebook is a simple deep CNN that was extracted from [the Keras examples](https://github.com/keras-team/keras/blob/master/examples/cifar10_cnn.py)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The dataset\n",
+    "The [CIFAR-10 dataset](https://www.cs.toronto.edu/~kriz/cifar.html) is one of the most popular machine learning datasets. It consists of 60,000 32x32 images belonging to 10 different classes (6,000 images per class). Here are the classes in the dataset, as well as 10 random images from each:\n",
+    "\n",
+    "![cifar10](https://maet3608.github.io/nuts-ml/_images/cifar10.png)\n",
+    "\n",
+    "In this tutorial, we will train a deep CNN to recognize these images.\n",
+    "\n",
+    "We'll compare file mode, pipemode datasets and distributed training with Horovod"
```

Comment:
Suggested:

"We'll compare using file mode and pipe mode for distributed training with Horovod."